test_rmw_implementation/test/test_publisher.cpp

```diff
@@ -245,8 +252,123 @@ TEST_F(CLASSNAME(TestPublisherUse, RMW_IMPLEMENTATION), get_actual_qos) {
   rmw_qos_profile_t actual_qos_profile = rmw_qos_profile_unknown;
   rmw_ret_t ret = rmw_publisher_get_actual_qos(pub, &actual_qos_profile);
   EXPECT_EQ(RMW_RET_OK, ret) << rmw_get_error_string().str;
-  EXPECT_EQ(rmw_qos_profile_default.history, actual_qos_profile.history);
-  EXPECT_EQ(rmw_qos_profile_default.depth, actual_qos_profile.depth);
-  EXPECT_EQ(rmw_qos_profile_default.reliability, actual_qos_profile.reliability);
-  EXPECT_EQ(rmw_qos_profile_default.durability, actual_qos_profile.durability);
+  EXPECT_EQ(qos_profile.history, actual_qos_profile.history);
+  EXPECT_EQ(qos_profile.depth, actual_qos_profile.depth);
+  EXPECT_EQ(qos_profile.reliability, actual_qos_profile.reliability);
+  EXPECT_EQ(qos_profile.durability, actual_qos_profile.durability);
+}
+
+TEST_F(
+  CLASSNAME(TestPublisherUse, RMW_IMPLEMENTATION),
```

Comment:
This alignment is strange since it aligns with the code. Is there not a way to adjust the indentation, or move the curly brace to a new line?

This happens elsewhere as well.

Reply:
That's `uncrustify`... I ended up shortening the test case name in https://github.com/ros2/rmw_implementation/pull/119/commits/da381880ea605ff00ee45481e64b1759f2d57594.